src/script/run-make.sh

```diff
@@ -73,6 +74,7 @@ function prepare() {
     elif type apt-get > /dev/null 2>&1 ; then
         install_cmd="apt-get install -y"
         which_pkg="debianutils"
+        compiler_pkg="clang-10"
```

Comment:
why? i think "clang" depends on "clang-10", and "clang" is more future-proof. on debian sid, "clang" depends on "clang-11", on ubuntu hirsute, "clang" depends on "clang-12".